code/modules/jobs/job_types/_job.dm

```diff
@@ -206,14 +206,14 @@
 		SSticker.OnRoundstart(CALLBACK(GLOBAL_PROC, .proc/_addtimer, CALLBACK(pick(GLOB.announcement_systems), /obj/machinery/announcement_system/proc/announce, "NEWHEAD", H.real_name, H.job, channels), 1))
 
 //If the configuration option is set to require players to be logged as old enough to play certain jobs, then this proc checks that they are, otherwise it just returns 1
-/datum/job/proc/player_old_enough(client/C)
-	if(available_in_days(C) == 0)
+/datum/job/proc/player_old_enough(client/player)
+	if(available_in_days(player) == 0)
```

Comment:
```suggestion
	if(!player || !available_in_days(player))
```
Clients can be nulled at any point for the first part. Regarding the == 0, I leave that up to you since on first blush it seems unnecessary, but it's possible it is needed in this specific case